code/modules/assembly/health.dm

```diff
@@ -21,7 +21,7 @@
 /obj/item/device/assembly/health/toggle_secure()
 	secured = !secured
 	if(secured && scanning)
-		SSobj.processing |= src
+		SSobj.processing.Add(src)
```

Comment:
Why Add instead of |=? You don't want the obj to be put twice in the processing list,even accidentally, an admin could set secured to 0 manually, or a future coder could do that and it'd add the object to the list twice